cluster-provision/gocli/cmd/run.go

```diff
@@ -596,6 +601,17 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			}
 		}
 
+		if psaEnabled {
+			success, err := docker.Exec(cli, nodeContainer(prefix, nodeName), []string{"/bin/bash", "-c", "ssh.sh sudo /bin/bash < /scripts/psa.sh"}, os.Stdout)
```

Comment:
aint this success masks the success of line 
https://github.com/kubevirt/kubevirtci/pull/915/commits/cdc94d85b81f82242afffdcd0c635a520ba61191#diff-12e7b6e3d4ce92e7d50b6cd0e3aebac95e9480cb4dbbe4a220e1c86c3b12f6ebR591

which determines if to run node01.sh or nodes.sh ?
https://github.com/kubevirt/kubevirtci/pull/915/commits/cdc94d85b81f82242afffdcd0c635a520ba61191#diff-12e7b6e3d4ce92e7d50b6cd0e3aebac95e9480cb4dbbe4a220e1c86c3b12f6ebR616